components/ClaimStatus.tsx

```diff
@@ -2,20 +2,24 @@ import { useTranslation } from 'next-i18next'
 
 import { NextSteps } from './NextSteps'
 import { TextLine } from './TextLine'
+import { TransLine } from './TransLine'
```

Comment:
Does that add clarity or just introduce HTML as an alias for the Trans component, since this is just a wrapper function?

Reply:
Ah I meant that it would be clearer to have Html as an alias for Trans, since HtmlLine vs TextLine is intuitively understandable (one is for pure text, the other when there's html) whereas HtmlLine vs TransLine is not unless you already know the i18next nomenclature.